sources/hasc.json

```diff
@@ -1,10 +1,12 @@
 {
-    "id": 1108827445,
-    "fullname": "Statoids HASC",
-    "name": "hasc",
-    "prefix": "hasc",
-    "key":"",
-    "url": "http://www.statoids.com/ihasc.html",
-    "license": "http://www.statoids.com/ihasc.html",
-    "description": "CC0 per email with Gwillim Law of Statoids on August 11, 2015: 'Yes. As far as I'm concerned, HASC codes are in the public domain - to encourage people or organizations to use them for data communication.'"
+	"id": 1108827445,
+	"fullname": "Statoids HASC",
+	"name": "hasc",
+	"prefix": "hasc",
+	"key": "",
+	"url": "http://www.statoids.com/",
+	"license": "http://www.statoids.com/ihasc.html",
+	"license_type": "CC0",
+	"license_text": "CC0 per email with Gwillim Law of Statoids on August 11, 2015: 'Yes. As far as I'm concerned, HASC codes are in the public domain - to encourage people or organizations to use them for data communication.",
```

Comment:
Move email comments starting with `Yes` to `description` (repeating the date and email bits).